packages/react-sdk/src/store/api/powerLevelsApi.ts

```diff
@@ -96,6 +99,65 @@ export const powerLevelsApi = baseApi.injectEndpoints({
       },
     }),
 
+    getCreateEvent: builder.query<
```

Comment:
Could you please update `powerLevelsApi.ts` with some test that create event is observed?

Also please update `usePowerLevels.test.tsx` with some room version 12 relevant test.